scripts/pre_redeploy.sh

```diff
@@ -107,3 +107,11 @@ popd
 pushd ${RPCO_DEFAULT_FOLDER}/rpcd/playbooks
     openstack-ansible configure-apt-sources.yml
 popd
+# Fix hostname with fixed period mark
+if [ -f /etc/ansible/roles/openstack_hosts/templates/openstack-host-hostfile-setup.sh.j2 ]; then
```

Comment:
Minor nit, wrap in double brackets and put file name in quotes to match reset of formatting in file.

Reply:
@antonym Is the request for double brackets coming from a particular stylistic preference or just defensive practice?  I'm asking because it touches on an outstanding question I have.  Do we, as a team, have any consistent style guides or best practices defined (formally or informally)?  It's not a huge deal, but when nits like this come up, it's really nice to be able to point to documentation.

Reply:
Mainly it came from the rest of the file was using if [[ ]] checks so I mentioned it more for consistency.  The quotes jumped more out at me.  I don't think there's really any formal guidelines in place at this time.